ds-prep-python-variables/main.py

```diff
@@ -0,0 +1,38 @@
+first_name = 'Gene'
+last_name = "Sun"
+full_name = first_name + " " + last_name
+print(full_name)
+
+height_in_inches = 70
+print("height_in_inches:", height_in_inches, type(height_in_inches))
+
+height_in_inches_float = float(height_in_inches)
+print("height_in_inches_float:", height_in_inches_float, type(height_in_inches_float))
+
+
+height_in_meters = (height_in_inches_float*2.54)/100
+print("height_in_meters:", height_in_meters)
+
+
+eats_plants = True
+eats_animals = False
+
+is_animal = (eats_plants) or (eats_animals)
+is_omnivore = (eats_plants) and (eats_animals)
+is_plant = not(is_animal)
+print("is_animal:", is_animal , "is_omnivore:", is_omnivore , "is_plant:", is_plant)
+
+
+mean_height_in_meters = 1.7155
+short_threshold_in_meters = 1.576
+tall_threshold_in_meters = 1.860
+
+is_mean_height = height_in_meters == mean_height_in_meters
+is_short = height_in_meters < short_threshold_in_meters
+is_tall = height_in_meters > tall_threshold_in_meters
```

Comment:
`is_tall` should also be `True` if `height_in_meters` is equal to `tall_threshold_in_meters`:

---

![image](https://user-images.githubusercontent.com/16143771/138135037-295a4bc4-95d8-443b-8f48-abccc8269c3c.png)